cpp/tests/test_abm_location.cpp

```diff
@@ -497,4 +497,26 @@ TEST(TestLocation, setCapacity)
     location.set_capacity(4, 200);
     ASSERT_EQ(location.get_capacity().persons, 4);
     ASSERT_EQ(location.get_capacity().volume, 200);
+}
+
+TEST(TestLocation, saveCurrentSubpopulations)
+{
+    auto location = mio::abm::Location(mio::abm::LocationType::PublicTransport, 0, 3);
+    auto person1  = mio::abm::Person(location, mio::abm::InfectionState::Infected, mio::abm::AgeGroup::Age5to14, {});
+    location.add_person(person1);
+    auto person2 = mio::abm::Person(location, mio::abm::InfectionState::Infected, mio::abm::AgeGroup::Age15to34, {});
+    location.add_person(person2);
+    auto person3 = mio::abm::Person(location, mio::abm::InfectionState::Exposed, mio::abm::AgeGroup::Age35to59, {});
+    location.add_person(person3);
+
+    auto t1 = mio::abm::TimePoint(7);
+    location.add_subpopulations_to_timeSeries(t1);
+
```

Comment:
Maybe add a change to the subpopulations between the two time points to check that everything works as intended.

Reply:
New test is added for abm_location.